make/modules/java.instrument/Lib.gmk

```diff
@@ -31,17 +31,29 @@ include LibCommon.gmk
 ## Build libinstrument
 ################################################################################
 
+ifeq ($(call isTargetOs, bsd), true)
```

Comment:
Once again, this is not how this works anymore. If you really need a static library, you need to change `BUILD_LIBJLI_TYPE` in make/modules/java.base/lib/CoreLibraries.gmk, as is done for AIX.